mhkit/wave/contours.py

```diff
@@ -1504,12 +1543,14 @@ def _bivariate_KDE(x1, x2, bw, fit, nb_steps, Ndata_bivariate_KDE, kwargs):
         max_x1 = x1.max()*2
     if isinstance(max_x2, type(None)):
         max_x2 = x2.max()*2
```

Comment:
@ssolson I'm not familiar with this method, but I noticed this line doesn't seem to match the description. Is it accurate to set ``max_x1 = 2*x1.max()``?

Reply:
This comes directly from WDRT:

https://github.com/WEC-Sim/WDRT/blob/71b2b871d27950bf14a944f432f8ee1221ee05e3/WDRT/ESSC.py#L2552-L2585
![image](https://github.com/MHKiT-Software/MHKiT-Python/assets/13438942/e4a84a34-28e2-4cdb-9b56-eee7b333a3ef)